### PR TITLE
fix: default redirect on settings route

### PR DESCRIPTION
**Pull request by pavanjoshi914**
properly tested

### CODE REVIEW

File: frontend/src/routes.tsx [hunk 1]
```diff
@@ -102,7 +101,7 @@ const routes = [
       },
       {
         path: "settings",
-        element: <SettingsLayout />,
```
Comment (rolznz) [inline frontend/src/routes.tsx:105]: Can you see anything wrong in the UI by doing this?

**Comment by rolznz** [comment]
@pavanjoshi914 can you fix the issue rather than just closing it?

File: frontend/src/routes.tsx (hunk 1 above)
Comment (pavanjoshi914) [reply frontend/src/routes.tsx:105]: should be fixed now

**Review by reneaaron — APPROVED** [review]
tACK

**Pull request merged**